code/game/objects/structures/mortar.dm

```diff
@@ -207,7 +207,27 @@
 		playsound(loc, 'sound/items/deconstruct.ogg', 25, 1)
 		new /obj/item/mortar_kit(loc)
 		qdel(src)
+	else if(istype(I,/obj/item/binoculars/tactical))
 
+		if(busy)
+			to_chat(user, "<span class='warning'>Someone else is currently using [src].</span>")
+			return
+		var/obj/item/binoculars/tactical/binoc = I
+		
+		if(binoc.target_turf.x == null || binoc.target_turf.y ==null)
```

Comment:
Don't you also need to have a check for whether an area is underground?

Reply:
can this stuff be underground?

Reply:
I mean as in do you need to have a check here for if the targetted turf is underground or not